x-pack/plugins/security_solution/public/alerts/components/alerts_table/alerts_filter_group/index.tsx

```diff
@@ -10,7 +10,11 @@ import * as i18n from '../translations';
 
 export const FILTER_OPEN = 'open';
 export const FILTER_CLOSED = 'closed';
-export type AlertFilterOption = typeof FILTER_OPEN | typeof FILTER_CLOSED;
+export const FILTER_IN_PROGRESS = 'in-progress';
+export type AlertFilterOption =
+  | typeof FILTER_OPEN
+  | typeof FILTER_CLOSED
+  | typeof FILTER_IN_PROGRESS;
```

Comment:
Is this a different type from the `Status` type we already have in the `schemas.ts`? It looks like it's the same type which is `open`, `closed`, `in-progress`. Can we just use the same `Status` so we don't have extra "touch points" when/if we update the names, types, etc...

I would also declare line 13 as not a string type but rather directly on create the enumeration type to make things easier.

Reply:
> I would also declare line 13 as not a string type but rather directly on create the enumeration type to make things easier.

what do you mean by this? mind giving an example?

Reply:
```ts
export const FILTER_IN_PROGRESS: Status = 'in-progress';
```

Does that work?

Reply:
yep, changed 👍